requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==1.8.2
+-e git+https://github.com/coolharsh55/python-opensubtitles@f37074796ee72c7efe1aa3ec2d534b4299b8fd88#egg=python_opensubtitles-master
```

Comment:
```pip install python-opensubtitles``` gives the following error and fails.
Which is why I tried installing python-opensubtitles using the github repo directly.
```bash
Collecting python-opensubtitles
  Could not find a version that satisfies the requirement python-opensubtitles (from versions: )
No matching distribution found for python-opensubtitles
```